libraries/ESP8266WiFi/src/include/ClientContext.h

```diff
@@ -342,7 +342,7 @@ class ClientContext
                 last_sent = millis();
             }
 
-            delay(0); // from sys or os context
+            esp_schedule(); // from sys or os context
```

Comment:
I would also add `esp_yield()` because 
- that's what `delay(0)` does
- it does not panic when in ISR
- this last change was following an issue that was annoying to track

Reply:
Absolutely, thanks - done.